options/locale/locale_en-US.ini

```diff
@@ -169,9 +169,12 @@ repos = Repositories
 users = Users
 organizations = Organizations
 search = Search
+code = Code
 repo_no_results = No matching repositories have been found.
 user_no_results = No matching users have been found.
 org_no_results = No matching organizations have been found.
+code_no_results = No matching code have been found.
```

Comment:
„code have been found“ should be changed to „code has been changed“

Reply:
Why changed?

Reply:
Maybe I should use `No matching codes have been found.`

Reply:
@JonasFranzDEV @lafriks done.